modules/dqt/jsx/react.paginator.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
I don't think you're using this file. The getJSDependencies is using the one from LORIS (`$baseURL . '/js/components/PaginationLinks.js'`)

Reply:
thanks @driusan! I'm unsure why I did that as some of the other components.